dist/libnccl-net-ofi.spec

```diff
@@ -0,0 +1,83 @@
+# Whether to build with cuda support. Default: on if neuron
+%if "%{with_cuda}" == "1" && "%{with_neuron}" == "1"
+%{error:Neuron and CUDA must not be enabled together}
+%endif
+
+%if "%{with_cuda}" == "0" && "%{with_neuron}" == "0"
+%{error:One of Neuron or CUDA must be enabled}
+%endif
+
+%if "%{with_cuda}" == "1"
+%{!?target: %global target nccl}
+%endif
+%if "%{with_neuron}" == "1"
+%{!?target: %global target nccom}
+%endif
+
+%global pname_base lib%{!?with_neuron:nccl}%{?with_neuron:nccom}-net-ofi
+%global pname %{pname_base}%{?with_platform_aws:-aws}
+
+%if "%{with_platform_aws}"
+%global _prefix /opt/amazon/%{pname_base}
+%endif
+
+# (CUDA only) what toolkit package to declare a build dependency on. Default: 12-6
+%{!?_cuda_toolkit_version: %global _cuda_toolkit_version 12-6}
+
+Name:           %{pname}
+Version:        1.11.0pre
```

Comment:
we should have this be dist/libnccl-net-ofi.spec.in and stamp the version out during configure.

Reply:
https://packit.dev/